data/lua/wml/kill.lua

```diff
@@ -50,19 +50,25 @@ function wesnoth.wml_actions.kill(cfg)
 			-- In other words, the attacker's weapon. The attacker is the secondary unit.
 			-- In the victory animation, this is simply swapped.
```

Comment:
This line about "the victory animation" appears to be a left-over from copying this from `[animate_unit]`'s implementation. Should it be removed?

Reply:
What makes you think it's a leftover? On line 76, the death animation is added, and on line 78, the victory animation is added with the animations swapped.

(Those are the line numbers as of 6ec5dcb8c4c9e904c8459e941f8a9509beba76db.)

Reply:
Somehow I missed line 78.